js/app/build_plugins.ts

```diff
@@ -192,7 +151,19 @@ export function handle_ce_css(): Plugin {
 					)
 			);
 
-			writeFileSync(html_location, transformed_html);
+			const share_html_location = join(config.dir, "share.html");
+			const share_html = readFileSync(share_html_location, "utf8");
+			const share_tree = parse(share_html);
+			const node = Array.from(
+				share_tree.querySelectorAll("link[rel=stylesheet]")
+			).find((node) => /.*\/index(.*?)\.css/.test(node.attributes.href));
+
+			if (!node) return;
+			const transformed_html =
+				share_html.substring(0, node.range[0]) +
+				share_html.substring(node.range[1], share_html.length);
+
+			writeFileSync(share_html_location, transformed_html);
```

Comment:
We are just stripping out the css file for the `share.html`, to prevent network errors. The CSS is loaded by the JS later anyway, so its no big deal.